scripts/switch-blue-green.sh

```
@@ -0,0 +1,213 @@
+        echo -e "${GREEN}[✓]${NC} ${env^} backend je dostupný (port $port_backend)"
+    else
+        echo -e "${RED}[✗]${NC} ${env^} backend NENÍ dostupný!"
+        return 1
+    fi
+    
+    return 0
+}
+
+# Funkce pro zobrazení statusu
+show_status() {
+    local current_env=$(get_current_env)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)